### PR TITLE
Add example-extractor annotation to Chapters 11, 12, and 13

### PR DESCRIPTION
**Pull request by RexJaeschke**

### CODE REVIEW

File: standard/expressions.md
Comment (jskeet) [inline standard/expressions.md]: Presumably this doesn't pass, so please give it an appropriate "problem" prefix, e.g. IncompleteExample.

File: standard/expressions.md
Comment (jskeet) [inline standard/expressions.md]: And again - I won't note further ones, but basically I'd like the shell script to pass on your machine before we merge.

I'd also note that if we're expecting output, standalone-lib is the wrong template :)

File: standard/expressions.md
Comment (jskeet) [inline standard/expressions.md]: Note to self: it would be lovely to be able to say "derive the expected output from the subsequent console output section of the spec". Once all the examples are *working*, we can see whether we're consistently using the same fencing etc.

File: standard/statements.md
Comment (jskeet) [inline standard/statements.md]: Ignore the warning? (Once everything is in, we should probably perform an audit that all the expected warnings correspond to descriptions in the text.)

File: standard/statements.md
Comment (jskeet) [inline standard/statements.md]: This is expecting each value on a different line - I would expect this to fail at the moment.

File: standard/statements.md
Comment (jskeet) [inline standard/statements.md]: There's no output from this example, because it's writing to a text file.

**Review by jskeet — COMMENTED** [review]
As with the other PR, let's make sure all the tests pass before merging. (Soon we should add a GitHub action that runs the tests.)

File: standard/statements.md
Comment (RexJaeschke) [reply standard/statements.md]: agreed

File: standard/expressions.md
Comment (RexJaeschke) [reply standard/expressions.md]: agreed

File: standard/expressions.md
Comment (RexJaeschke) [reply standard/expressions.md]: agreed

File: standard/statements.md
Comment (RexJaeschke) [reply standard/statements.md]: fixed

File: standard/statements.md
Comment (RexJaeschke) [reply standard/statements.md]: agreed

**Comment by jskeet** [comment]
@RexJaeschke: Please review the final commit. I've used "UsingsExample" for where we're missing using directives. (Effectively that's a subset of IncompleteExample, I guess.)

**Review by jskeet — APPROVED** [review]
With my final commit in place, I'm happy to merge this when Rex is happy.

**Comment by RexJaeschke** [comment]
@jskeet I'm happy with this.

**Pull request merged**
